app/views/users/show.html.erb

```diff
@@ -1,36 +1,45 @@
 <section>
-
-  <div class='user'>
-    <div class='user_profile'>
-      <img src=<%= @user.Photo %> alt="<%=@user.Name%>'s image" />
+  <div >
+    <div >
+      <img src="<%= @user.Photo %>" alt="image">
     </div>
-    <div class='user_info'>
-      <h4 class='user_name'><%= @user.Name %></h4>
-      <p class='user_activity'>Number of posts: <%= @user.posts.count %></p>
+
+    <div>
+      <h2>
+      <%= @user.Name %></h2>
+        <p>
+        <span>Number of posts: </span>
+        <%= @user.PostsCounter || 0 %>
+        </p>
+        
     </div>
   </div>
 
-  <div class='user_bio'>
-    <h4>Bio</h4>
-    <p><%= @user.Bio %></p>
+  <div>
+    <h4> Bio </h4>
+    <p>
+      <%= @user.Bio %>
+    </p>
```

Comment:
- [ ] I was not able to view the posts because I can't access the `users#show` page. Here is the screenshot showing the error 👇 
![image](https://user-images.githubusercontent.com/90222110/216621724-35f20c54-304c-444d-ba31-6155b668dc06.png)

Please fix the issue too 😃 👍